src/components/Tab/ValidityIndicator/validityIndicator.scss

```diff
@@ -0,0 +1,26 @@
+.validity-indicator {
+  width: 32px;
```

Comment:
It looks perfect for the two digits but feels empty with one digit. Let's do the size dynamic
![image](https://github.com/weka/weka-ui-components/assets/116369607/410aad2e-229c-4a2e-83f8-fe533a60b8cd)

Reply:
I think it's better to have all the same size
it's look aligned that way